sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/core/SqlDelightPropertiesFile.kt

```diff
@@ -47,7 +48,8 @@ data class SqlDelightDatabaseProperties(
   val compilationUnits: List<SqlDelightCompilationUnit>,
   val outputDirectory: String,
   val className: String,
-  val dependencies: List<SqlDelightDatabaseName>
+  val dependencies: List<SqlDelightDatabaseName>,
+  val dialectPreset: DialectPreset
```

Comment:
If you update the IDE plugin but not the Gradle plugin this will be null and you'll get a crash. Add a default?

Reply:
👍